<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminCatalogue</class>
 <widget class="QMainWindow" name="adminCatalogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: #FFE5D0;
	font-family: Montserrat;
}

QLineEdit {
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
	font-size: 13px;
}

QPushButton {
	border-radius: 15px; 
	font-size: 13px;
	height: 31px; 
}</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>1081</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
	border: none;
}
</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1081</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #FFE5D0;
	/*border: none;*/
}

QGridLayout {
	/*border: none;*/
}

QPushButton {
	margin-right: 10px; 
	height: 31px; 
	/*max-width: 100px; */
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: none;
}

QPushButton {
	color: #E78A6B;
	font-weight: 500;
}

QPushButton:hover {
	 text-decoration: underline; 
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="logoImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/resources/images/bblLogo.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="catalogueButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Catalogue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="membersButton">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>31</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Members</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>999</x>
       <y>31</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>delete record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>print record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="catalogueButton_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Book</string>
     </property>
    </widget>
    <widget class="QPushButton" name="membersButton_2">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>30</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Member</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="savePushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>105</y>
      <width>1081</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(0,0,0,0.1);
border-radius: 1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
